src/extension/features/toolkit-reports/pages/income-vs-expense/components/monthly-transaction-totals-table/component.tsx

```diff
@@ -1,22 +1,23 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import { CollapsibleRow } from './components/collapsable-row';
-import { MonthlyTotalsRow } from 'toolkit-reports/pages/income-vs-expense/components/monthly-totals-row';
+import { MonthlyTotalsRow } from 'toolkit/extension/features/toolkit-reports/pages/income-vs-expense/components/monthly-totals-row';
 import './styles.scss';
+import { NormalizedExpenses, NormalizedIncomes } from '../../types';
 
 export const TableType = {
   Expense: 'expense',
   Income: 'income',
-};
+} as const;
```

Comment:
We seem to prefer `enum` for these kinds of things (although personally I like this better).